insights/tests/client/test_client.py

```diff
@@ -1,10 +1,34 @@
 import sys
+import os
+import pytest
+import time
 
 from insights.client import InsightsClient
 from insights.client.config import InsightsConfig
 from insights import package_info
-from mock.mock import patch
-from mock.mock import Mock
+from insights.client.constants import InsightsConstants as constants
+# from mock.mock import patch
+# from mock.mock import Mock
+
+
+class FakeConnection(object):
```

Comment:
I’d prefer to use _Mock_ instead. It’s not 100% necessary, but in the tests already present (introduced by my PR #1251) it‘s already being mocked/patched, so we can stick to that.